homeassistant/components/modbus/const.py

```diff
@@ -82,5 +83,10 @@
     DATA_TYPE_UINT: {1: "H", 2: "I", 4: "Q"},
     DATA_TYPE_FLOAT: {1: "e", 2: "f", 4: "d"},
 }
+DEFAULT_STRUCT_FORMAT_BYTES = {
```

Comment:
That is basically the same as DEFAULT_STRUCT_FORMAT, that looks wrong, unless there are a good reason in the code these 2 structures needs to be unified.